### PR TITLE
[WIP] Add a failing test for fela-monolithic usage with React

### PR DESCRIPTION
**Pull request by vlad-zhukov**
This is a repro of #395

### CODE REVIEW

File: packages/fela-monolithic/src/__tests__/__snapshots__/integration-react-test.js.snap [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Monolithic enhancer React integration should render a component 1`] = `".f137u7ef{color:red}.f137u7ef{color:red}"`;
```
Comment (vlad-zhukov) [inline packages/fela-monolithic/src/__tests__/__snapshots__/integration-react-test.js.snap]: There should be a single class like it is in v5.

**Comment by vlad-zhukov** [comment]
It shouldn't be merged 😄

**Comment by rofrischmann** [comment]
Hum why not :P it's a valid test case I guess

**Comment by vlad-zhukov** [comment]
Because right now it outputs `.f137u7ef{color:red}.f137u7ef{color:red}` instead of `.f137u7ef{color:red}`. The CI is failing due to lerna bootstrap error. I should've also be using equality test instead of snapshotting.

**Comment by rofrischmann** [comment]
snapshot is totally fine, we just need to fix it first, but I already got that working I guess

**Pull request merged**